### ISSUE TITLE
.NET 8 Changing BackgroundColor through an attached property does not work on Android 

### ISSUE THREAD

**Issue by FreakyAli** [dotnet/maui#19576]
### Description

I have a Static class that uses attached properties to create a shimmer effect, This stops updating the BackgroundColor with .NET 8. Works fine on iOS and on both with .NET 7 and 6.

### Steps to Reproduce

- Go to the repo link below and download.
- Run and visit the skeleton effect screen.
- The correct background colour will update on iOS once the skeleton effect is complete but fails on Android. (The colour is assigned correctly yet it does not do anything)

| .NET 8 | .NET 7 |
| --- | --- |
| <img src="https://github.com/dotnet/maui/assets/31090457/a2398bcf-99e3-4c49-86b3-21a19cb881db" width="250" height="550"/>| <img src="https://user-images.githubusercontent.com/31090457/225077308-994f0265-463a-4109-bca1-8abc0cf634c4.gif" width="250" height="550"/> |

### Link to public reproduction project repository

https://github.com/FreakyAli/Maui.FreakyEffects

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI, Yes, this used to work in Xamarin.Forms

### Last version that worked well

7.0.101

### Affected platforms

Android

### Affected platform versions

All

### Did you find any workaround?

No

**Comment by FreakyAli** [comment]
Any updates guys my plugin is pretty much dependent on it!?

**Comment by QianaJiao** [comment]
Verified this issue with Visual Studio 17.10.0 Preview 1; Maui 8.0.7; I can repro on Android platform with Sample project.
![image](https://github.com/dotnet/maui/assets/91244512/c6b58ee8-e6c0-4bf6-8a06-f82aeb6c1049)

**Comment by FreakyAli** [comment]
@jfversluis Can we take a look into this one, Its an important feature and is not working on Android

**Comment by jfversluis** [comment]
Feel free to have a look 😉

**Comment by FreakyAli** [comment]
@QianaJiao Can we do something about this, I have a plugin that's dependent on it!

**Comment by Jon2G** [comment]
Spent hours dealing with this, its affecting lots of plugins!

**Comment by FreakyAli** [comment]
@Jon2G did you find some kind of a work around to it?

**Comment by Jon2G** [comment]
I'm downgrading to net 7 
Awful 😔

**Comment by FreakyAli** [comment]
@Jon2G I thought .NET7 is unsupported now...

**Comment by FreakyAli** [comment]
any updates here?

**Comment by FreakyAli** [comment]
still not working with .NET9

**Comment by FreakyAli** [comment]
@PureWeen  Do you see this coming with a .NET 10 update anytime soon?

**Comment by FreakyAli** [comment]
Yes I've tried reproducing this every 2-3 release for the last 2+ years, so yeah I'm sure this is not working!

My plugin is literally bricked all thanks to this bug! The last version I tried with is 10.0.90

**Comment by kubaflo** [comment]
Thanks for confirming the current Android behavior. The issue is open and the revalidation label is no longer present, so it will remain available for investigation.

> Disclosure: this comment was generated by AI.
